projects/aca-content/src/lib/store/effects/search.effects.ts

```diff
@@ -49,14 +52,13 @@ export class SearchEffects {
       this.actions$.pipe(
         ofType<SearchByTermAction>(SearchActionTypes.SearchByTerm),
         map((action) => {
-          const query = action.payload.replace(/%/g, '%25').replace(/[(]/g, '%28').replace(/[)]/g, '%29');
-
+          const query = formatSearchTerm(action.payload, this.queryBuilder.config['app:fields']);
           const libItem = action.searchOptions.find((item) => item.id === SearchOptionIds.Libraries);
           const librarySelected = !!libItem && libItem.value;
           if (librarySelected) {
```

Comment:
Can be shorter: 
this.queryBuilder.navigateToSearch(query, librarySelected ? ‘/search-libraries’ : ‘/search’);